lib/axiosHttpClient.ts

```diff
@@ -13,7 +13,7 @@ import { WebResource, HttpRequestBody } from "./webResource";
 
 const axiosClient = axios.create();
 // Workaround for https://github.com/axios/axios/issues/1158
-axiosClient.interceptors.request.use(config => ({ ...config, method: config.method && config.method.toUpperCase() }));
+axiosClient.interceptors.request.use(config => ({ ...config, method: config.method && config.method.toUpperCase() as any }));
```

Comment:
`npm run local` actually caused me to use my local version of axios, which brought in new, stricter type definitions that required this change.